hil-test/src/bin/esp_radio_init.rs

```diff
@@ -133,7 +164,13 @@ mod tests {
     fn test_esp_preempt_sleep_wakes_up(p: Peripherals) {
         use esp_radio_preempt_driver as preempt;
         let timg0 = TimerGroup::new(p.TIMG0);
-        esp_preempt::start(timg0.timer0);
+        esp_preempt::start(
+            timg0.timer0,
+            #[cfg(riscv)]
```

Comment:
I'm wondering how bad this asymmetry in the API is. The fact is, Xtensa doesn't require a FROM_CPUn interrupt to drive a single-core scheduler, while risc-v does not seem ti have any other software-triggerable, core-specific interrupts (i.e. an analogous mechanism to `wsr.intset`)

Reply:
it's not really nice, but for C2/C3 I don't see another way